orbstation/modules/client/preferences/height_pref.dm

```diff
@@ -0,0 +1,23 @@
+//character preference, makes character height selectable from five options
+
+/datum/preference/choiced/height
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "height"
+	can_randomize = FALSE
+	var/height_list = list(
+		"Shortest" = HUMAN_HEIGHT_SHORTEST,
+		"Short" = HUMAN_HEIGHT_SHORT,
+		"Medium" = HUMAN_HEIGHT_MEDIUM,
+		"Tall" = HUMAN_HEIGHT_TALL,
+		"Tallest" = HUMAN_HEIGHT_TALLEST,
+	)
+
+/datum/preference/choiced/height/init_possible_values()
+	return list("Shortest", "Short", "Medium", "Tall", "Tallest")
```

Comment:
I don't know if this exists in byond but if there's a way to do like `return height_list.keys` so it automatically matches the keys of the list above that would be ideal. Not hugely necessary if it's annoying to do though.

Reply:
For future reference, assoc_to_keys(list) is how you do this. So, it's super easy.